css/main.css

```diff
@@ -261,65 +273,100 @@ section#credits h1 {
   color: white;
 }
 
-ul.credits-column {
-  list-style: none;
-  padding: 0;
-  margin: 0;
-  
-  box-sizing: border-box;
-  text-align: center;
-
+section.credits-column {
   min-height: auto;
   padding: 1.5em;
-  background: #BBD5DA;
+  background: #bbd5da;
   border-top-left-radius: 0.5em;
   border-bottom-right-radius: 0.5em;
-  margin: 1.5em 0.5em;
+  margin-left: 0.5em;
+  margin-right: 0.5em;
+  margin-top: 0;
+  margin-bottom: 1.5em;
   /* avoid breaking the message box between columns */
   break-inside: avoid;
 
   animation: 1s slide-up ease;
   animation-fill-mode: both;
   animation-iteration-count: 1;
 }
+section.credits-column ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
 
-ul.credits-column:nth-of-type(1) { animation-delay: .5s; margin-top: 0; }
-ul.credits-column:nth-of-type(2) { animation-delay: 1s }
-ul.credits-column:nth-of-type(3) { animation-delay: 1.5s }
+  box-sizing: border-box;
+  text-align: center;
+}
 
-ul.credits-column li {
+section.credits-column:nth-of-type(1) {
+  grid-area: general;
+  animation-delay: 0.5s;
+  margin-top: 0;
+}
+section.credits-column:nth-of-type(2) {
+  grid-area: devs;
+  animation-delay: 1s;
+}
+section.credits-column:nth-of-type(3) {
+  grid-area: twitillust;
+  animation-delay: 1.5s;
+}
+
+section.credits-column li {
   margin: 2em 0;
 }
+section.credits-column#twitter-images {
+  grid-area: twitter;
+}
+section.credits-column#twitter-images h2 {
+  text-align: center;
+}
+section.credits-column#twitter-images ul {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-column-gap: 20px;
+}
 
-ul.credits-column h2 {
+section.credits-column h2 {
   font-size: 1.25em;
   margin-bottom: 0.25em;
 }
 
-ul.credits-column h3 {
+section.credits-column h3 {
   font-size: 1em;
   margin: 0.5em 0;
 }
 
-ul.credits-column a {
-  color: var(--bg-color)
+section.credits-column a {
+  color: var(--bg-color);
 }
 
-ul.credits-column a:visited {
+section.credits-column a:visited {
   color: var(--bg-color-lighten-50);
 }
 
-ul.credits-column p {
+section.credits-column p {
   margin-top: 0.25em;
   font-size: 1em;
 }
 
+.credits-page-artwork {
+  height: auto;
+  width: 100%;
+}
```

Comment:
main change - specify dimensions of the imgs in the twitter art section. width: 100% will follow what's specified by css grid

Reply:
changed to use flex instead per Kiri's suggestion in discord thread